ruby/basics/conditional_logic.md

```diff
@@ -1,43 +1,246 @@
+    * When should you use a case statement?
 
 ## Truthy and Falsy in Ruby
-- Brief explanation what ruby condiders to be true and what ruby considers to be false. 
-- The only two things in ruby which are false is nil and false
+
+You already know that conditional statements check expressions for a true or false value so it follows that you should understand what Ruby considers to be true or false. In typical Ruby fashion it's very simple. The only false values in ruby are the values `nil` and `false` itself. That's it! Everything else is considered true. Even the string `"false"` is true in conditional expressions! If you have experience with other programming languages you might be familiar with the number 0 or an empty string "" being equivalent to false, this isn't the case with Ruby so be careful when writing those expressions otherwise you'll have more bugs than a decomposing body.
 
 ## Basic Conditional Statement
-- How to create a single branch if statement
-- single line if statements
+The simplest way to control the flow of your code using conditions is with the `if` statement.
+
+The general syntax is
+
+```ruby
+if (statement_to_be_evaluated == true)
+  do something awesome...
+end
+
+if 1 < 2
+  puts "Hot diggity damn, 1 is less than 2"
+end
+```
+
+If there is only one line of code to be evaluated inside the block then you can rewrite the example above as follows
+```ruby
+puts "Hot diggity damn, 1 is less than 2" if 1 < 2
+```
+You don't even need the `end` statement. Nice and concise!
+
+The statement to be evaluated can be anything that returns true or false. It could be a mathematical expression, a variable value or a call to a method. Only if the expression evaluates to true does the code inside the block run.
 
 ## Branching in if statements
-- How to create different branches in conditional statments and which one will be chosen.
-- elsif if
+Often you'll want to check a condition and run some code if it's true, but if it's false then run some other code. That's done with an `if; else` statement.
+
+```ruby
+if (attack_by_land == true)
+  puts "release the goat"
+else
+  puts "release the shark"
+end
+```
+Oh yeah! Protected on land and sea.
+
+But what about if we're attacked by air? We need yet another conditional check. Enter the `if; elsif; else` statement
+
+```ruby
+if (attack_by_land == true)
+  puts "release the goat"
+elsif (attack_by_sea == true)
+  puts "release the shark"
+else
+  puts "release Kevin the octopus"
+end
+```
+Ain't nobody pillaging my land!
+
+You can have as many elsif expressions as you want and the else clause is optional but you usually want to provide some default value if none of the previous expression evaluate to true.
+
+## Boolean Logic
+To use an expression that evaluates to true or false you'll need a comparison operator. There are several provided by Ruby and in no particular order they are:
+
+`==`(equals) returns true if both of the values compared are equal.
+`5 == 5 #=> true`
```

Comment:
I think we should provide opposing examples for boolean operators too.
for example
```
5 == 5 #=> true
5 == 6 #=> false
```

Just to get the idea across